.github/workflows/scheduled.yaml

```diff
@@ -39,3 +39,18 @@ jobs:
               "tag": "",
               "tag_latest": "true"
             }
+      - name: Build an image from Dockerfile
+        run: |
+          docker build -t ${{ matrix.image.distro }}:${{ github.sha }} .
```

Comment:
Nope. Please do not do this. This is extremely wasteful.

Simply fetch the Docker image from the registry instead.

Reply:
done